controllers/pwas/crud.js

```diff
@@ -126,12 +131,13 @@ router.post(
  *
  * Display a PWA.
  */
-router.get('/:pwa', function get(req, res, next) {
+router.get('/:pwa', (req, res, next) => {
   pwaModel.find(req.params.pwa, (err, entity) => {
     if (err) {
       return next(err);
     }
 
+    entity.firstletter = entity.name[0];
```

Comment:
I think it would be better to either do these entity transformations and enhancements either in a separate function, or with a [helper](http://handlebarsjs.com/#helpers) at render time--it's not really clear why this is happening here.